__tests__/model/car/Cars.test.js

```diff
@@ -0,0 +1,51 @@
+import Car from '../../../src/model/car/Car.js';
+import Cars from '../../../src/model/car/Cars.js';
+
+/* eslint-disable max-lines-per-function */
+describe('[단위 테스트] Cars', () => {
+  let cars;
+
+  beforeEach(() => {
+    cars = new Cars();
+  });
+
+  describe('[함수] makeCars', () => {
+    test('주어진 자동차 이름들로 자동차를 생성하는지 확인', () => {
+      const carNames = ['Car1', 'Car2', 'Car3'];
+      cars.makeCars(carNames);
+
+      const createdCars = cars.getCars();
+
+      expect(createdCars.length).toEqual(carNames.length);
+      expect(createdCars[0]).toBeInstanceOf(Car);
+      expect(createdCars[0].getName()).toEqual(carNames[0]);
```

Comment:
시도는 안해봤지만 forEach로 loop를 돌면서 expect 하는건 안되나요?? 
이렇게 검사하는 것보다 더 정확할 수도 있을 것 같은생각이 드네요.

Reply:
리뷰 내용에 동의합니다! `every`를 사용하여 리팩토링 해보았습니다.

```suggestion
      expect(createdCars.length).toEqual(carNames.length);
      expect(
        createdCars.every((car, index) => car instanceof Car && car.getName() === carNames[index])
      ).toBe(true);
```

Reply:
오 너무 좋습니다!
저는 every도 마찬가지로 깔끔하다고 생각하긴 합니다. 둘 다 좋아요!!